enterprise-suite/scripts/install-es.sh

```diff
@@ -152,13 +154,22 @@ else
     should_upgrade=false
 fi
 
+# OPENSHIFT_V_3.6 Support
+OPENSHIFT36_APIVERSIONS=
+if [ "openshift36" == "$ES_KUBERNETES" ]; then
+    OPENSHIFT36_APIVERSIONS="--set rbacApiVersion=authorization.openshift.io/v1,deploymentApiVersion=apps/v1beta1,daemonSetApiVersion=extensions/v1beta1"
+fi
```

Comment:
Do you think we'll need this? Or can we just document the versions to use?

Reply:
Removed changes to installer script